benchmarks/apps/ziatest/ziatest.py

```diff
@@ -0,0 +1,50 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+from benchmarks.modules.utils import SpackTest
+
+
+@rfm.simple_test
+class ZiatestBenchmark(SpackTest):
+    valid_systems = ['-gpu']
```

Comment:
I'm not sure about this one, I guess people may want to run the benchmark on any type of system.